docs/minio-client-complete-guide.md

```diff
@@ -534,6 +536,65 @@ mc head -n 1 --encrypt-key "play/mybucket=32byteslongsecretkeymustbegiven1" play
 Hello!!
 ```
 
+<a name="lock"></a>
+### Command `lock` - set and get object lock configuration
+`lock` sets and gets object lock configuration
+
+```
+USAGE:
+   mc lock [FLAGS] TARGET [governance | compliance] [VALIDITY]
+
+FLAGS:
+  --clear, -c                   clears previously stored object lock configuration
+  --json                        enable JSON formatted output
+  --help, -h                    show help
+```
+
+*Example: Set object lock configuration of 30 day compliance on bucket `mybucket`*
+
+```
+mc lock myminio/mybucket compliance 30d
+```
+
+*Example: Display the object lock configuration for bucket `mybucket`*
+
+```
+mc lock myminio/mybucket
+COMPLIANCE mode is enabled for 30d
+```
+*Example: Display the object lock configuration for bucket `mybucket`*
+
+```
+mc lock myminio/mybucket
+COMPLIANCE mode is enabled for 30d
+```
+*Example: Clear object lock configuration for bucket `mybucket`*
+
+```
+mc lock --clear myminio/mybucket
+No mode is enabled
```

Comment:
this looks like wrong error we may have to fix this its not clear what does it mean by `No mode is enabled`

Reply:
fixed this